targets/DeployDependencies.proj

```diff
@@ -20,17 +20,40 @@
           SkipUnchangedFiles="true"           
           />
   </Target>
+  
+  <!-- Copy content of BuildTools package to deployment folders -->
+  <Target Name="CopyBuildTools">
+    <PropertyGroup>
+      <BuildToolsVersionFile>$(RepoRoot)BuildToolsVersion.txt</BuildToolsVersionFile>
+    </PropertyGroup>
+    <ReadLinesFromFile File="$(BuildToolsVersionFile)">
+      <Output TaskParameter="Lines" ItemName="BuildToolsVersion" />
+    </ReadLinesFromFile>
+    <PropertyGroup>
+      <BuildToolsVersion>@(BuildToolsVersion)</BuildToolsVersion>
+    </PropertyGroup>
+    <ItemGroup>
+      <BuildToolsFiles Include="$(ToolPackagesDir)Microsoft.DotNet.BuildTools\$(BuildToolsVersion)\lib\*.*" />
```

Comment:
Is everything in there needed to be in the msbuild deployment folders? Or only a subset of those files?